editor/src/components/canvas/canvas-strategies/strategies/absolute-move-strategy.tsx

```diff
@@ -74,7 +74,7 @@ export function absoluteMoveStrategy(
       fitness:
         interactionSession?.interactionData.type === 'DRAG' &&
         interactionSession?.activeControl.type === 'BOUNDING_AREA'
-          ? 1
+          ? 2
```

Comment:
Hmmm this change makes me suspicious about whether it will cause issues with other strategies. I believe we have covered a ton of cases in the tests, so it should be fine, but if we suddenly start seeing other strategies failing to fire when we'd expect them to, we will want to check if this is the cause

Reply:
yeah I'm a bit on the verge as well, but it _looks_ like this would be fine. Maybe it's worth to add a comment here, but I wonder what it should say 🤔

Reply:
[`7755cdf` (#4298)](https://github.com/concrete-utopia/utopia/pull/4298/commits/7755cdf0b958030e92f095d3fc6ad582e79b611c)